morpheus/pipeline/stage.py

```diff
@@ -52,6 +52,9 @@ def on_start(self):
         """
         pass
 
+    async def start_async(self):
+        pass
+
```

Comment:
Can you add a docstring here that specifies that both `start_async` and `start` are called during stage initialization. The `start_async` allows stages to utilize the `asyncio` loop in their stages if needed.